source/retryable-reads/tests/estimatedDocumentCount-serverErrors.yml

```diff
@@ -1,9 +1,11 @@
 runOn:
     -
         minServerVersion: "4.0"
+        maxServerVersion: "4.9.0"
```

Comment:
Note, the [Test Format](https://github.com/mongodb/specifications/blob/master/source/retryable-reads/tests/README.rst#test-format) for Retryable Reads states that `maxServerVersion` is "inclusive". You should probably use something like "4.8.99" here if you want to exclude 4.9+.